litellm/proxy/proxy_server.py

```diff
@@ -2991,6 +2992,11 @@ def model_list(
     dependencies=[Depends(user_api_key_auth)],
     tags=["completions"],
 )
+@router.post(
+    "/openai/deployments/{model:path}/completions",
+    dependencies=[Depends(user_api_key_auth)],
+    tags=["completions"],
+)
```

Comment:
@krrishdholakia i guess this part is still needed if you use the azure client? it's not present in the dev release
with the query param part as well

Reply:
just added - https://github.com/BerriAI/litellm/commit/0342cd3b6b0aefb1b46e6d1b27248a403d8d7048

Reply:
what about the api version being passed as a query param?
is it handled somewhere else already?